nuclia/e2e/conftest.py

```diff
@@ -0,0 +1,300 @@
+
+import imaplib
+import email
+import re
+import time
+from email.header import decode_header
+import tempfile
+from nuclia.config import reset_config_file, set_config_file
+from nuclia.data import get_auth
+from nuclia.data import get_config
+import nuclia
+from nuclia.sdk.auth import NucliaAuth
+from nuclia.sdk.kbs import NucliaKBS
+from copy import deepcopy
+
+
+# All tests that needs some existing account will use the one configured in the global "permanent_account_slug"
+# with some predefined user credentials without expiration:
+# "permanent_account_owner_pat": PAT token for `testing_sdk@nuclia.com` on the suitable account
+
+CLUSTERS_CONFIG = {
+    "prod": {
+        "global": {
+            "base_url": "https://nuclia.cloud",
+            "recaptcha": os.environ.get("PROD_GLOBAL_RECAPTCHA"),
+            "root_pat_token": os.environ.get("PROD_ROOT_PAT_TOKEN"),
+            "permanent_account_slug": "automated-testing",
+            "permanent_account_owner_pat_token": os.environ.get("PROD_PERMAMENT_ACCOUNT_OWNER_PAT_TOKEN")
+        },
+        "zones": [
+            {
+                "name": "aws-us-east-2-1",
+                "zone_slug": "aws-us-east-2-1",
+                "base_url": "https://aws-us-east-2-1.nuclia.cloud",
+                "test_kb_slug": "nuclia-e2e-live-aws-us-east-2-1"
+            },
+            {
+                "name": "europe-1",
+                "zone_slug": "europe-1",
+                "base_url": "https://europe-1.nuclia.cloud",
+                "test_kb_slug": "nuclia-e2e-live-europe-1"
+            },
+        ]
+    },
+    "stage": {
+        "global": {
+            "base_url": "https://stashify.cloud",
+            "recaptcha": os.environ.get("STAGE_GLOBAL_RECAPTCHA"),
+            "root_pat_token": os.environ.get("STAGE_ROOT_PAT_TOKEN"),
+            "permanent_account_slug": "automated-testing",
+            "permanent_account_owner_pat_token": os.environ.get("STAGE_PERMAMENT_ACCOUNT_OWNER_PAT_TOKEN")
+        },
+        "zones": [
+            {
+                "name": "europe-1",
+                "zone_slug": "europe-1",
+                "base_url": "https://europe-1.stashify.cloud",
+                "test_kb_slug": "nuclia-e2e-live"
+
+            },
+            # uncomment to test two zone parallel testing on stage
+            # {
+            #     "name": "europe-2",
+            #     "zone_slug": "europe-1",
+            #     "base_url": "https://europe-1.stashify.cloud",
+            #     "test_kb_slug": "nuclia-e2e-live"
+            # }
+
+        ]
+    }
+}
+TEST_ACCOUNT_SLUG = "test-e2e-creation"
+TEST_ENV = os.environ.get("TEST_ENV")
+TEST_ONBOARD_INQUIRY = {
+    "company": "Nuclia e2e",
+    "use_case": "Other",
+    "role": "Other",
+    "organization_size": "1-50",
+    "phone": "+34 111 222 333",
+    "receive_updates": False
+}
+
+
+@pytest.fixture(scope="session")
+def test_config():
+    return {
+        "test_account_slug": TEST_ACCOUNT_SLUG
+    }
```

Comment:
I think this makes more sense to be imported in each test directly, instead of using a fixture

Reply:
same reasoning about the other one, i have no other reasons. Honestly this was an experiment, as usually i do the import thing, but i quite like getting rid of this kind of imports on the main test files as it isolates the tests from how you are organizing this settings, as you only rely on the fixture